app/controllers/granules_controller.rb

```diff
@@ -60,7 +60,17 @@ def pull_latest
     granule = Granule.find(params[:id])
     collection = granule.collection
     begin
-      Granule.add_new_revision_to_granule(collection.concept_id, granule, current_user)
+      granule_components = Granule.add_new_revision_to_granule(collection.concept_id, granule, current_user)
```

Comment:
This code is copied from the business case 'Add new random granule'  (See add_granule_with_concept_id in models/collection.rb)